person_class.rb

```diff
@@ -0,0 +1,19 @@
+# This is the main class of the application
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
```

Comment:
- [ ] [REQUIRED] Please add `attr_reader` in line 4 and give valure `id` and remove the `id` from `attr_accessor`!

Reply:
yes sure i will remove id from att_accessor. but why add att-reader ? the attr_accessor is the short for att_reader and att_writer